repos/system_upgrade/el7toel8/actors/forcedefaultboottotargetkernelversion/libraries/forcedefaultboot.py

```diff
@@ -40,6 +41,10 @@ def update_default_kernel(kernel_info):
     else:
         try:
             stdlib.run(['grubby', '--set-default', kernel_info.kernel_path])
+            if architecture.matches_architecture(architecture.ARCH_S390X):
+                # on s390x we need to call zipl explicitly because of issue in grubby,
```

Comment:
If there's a BZ for the grubby issue, link it here.

Reply:
I have been thinking about that when I wrote the comment, but it could be confusing because the bugzilla is about something different, and the problem is just mentioned there. So not sure how useful it would be for people:
  https://bugzilla.redhat.com/show_bug.cgi?id=1755573

Reply:
I think it's worth adding it there. Because if it's not, we wouldn't easily know whether the issue is fixed or not in order to decide whether we can remove this workaround.

Reply:
@bocekm My point is, that if the BZ issue is fixed, we cannot remove it because the BZ is about different problem. but ok. I will create new one, but I am almost sure that issue will never be fixed.

Reply:
Done. See: See: https://bugzilla.redhat.com/show_bug.cgi?id=1764306